github/issues_labels_test.go

```diff
@@ -160,7 +160,7 @@ func TestIssuesService_ListLabelsByIssue(t *testing.T) {
 	mux.HandleFunc("/repos/o/r/issues/1/labels", func(w http.ResponseWriter, r *http.Request) {
 		testMethod(t, r, "GET")
 		testFormValues(t, r, values{"page": "2"})
-		fmt.Fprint(w, `[{"name": "a"},{"name": "b"}]`)
+		fmt.Fprint(w, `[{"name": "a","id": 1},{"name": "b", "id": 2}]`)
```

Comment:
Nitpick. It's inconsistent how first added comma has no space after it, but the other comma has a space after it.